src/DoctrineV2MessageRepository/DoctrineUuidV4MessageRepositoryTest.php

```diff
@@ -22,7 +22,6 @@ class DoctrineUuidV4MessageRepositoryTest extends MessageRepositoryTestCase
     protected function setUp(): void
     {
         if ( ! interface_exists(ResultStatement::class)) {
-            die("WHAT");
```

Comment:
WHAT 😆

Reply:
lol, probably a left-over from my "challenges" with the multi-version setup.